modules/custom-status/rest/custom-status-endpoint.php

```diff
@@ -349,3 +387,5 @@ public static function get_reorder_url() {
 		return rest_url( sprintf( '%s/%s', VIP_WORKFLOW_REST_NAMESPACE, 'custom-status/reorder' ) );
 	}
 }
+
+CustomStatusEndpoint::init();
```

Comment:
I updated this endpoint and all others in our plugin to `::init()` themselves. There was really no reason to `::init()` manually in the including class.